hooks/post_gen_project.sh

```diff
@@ -26,7 +26,7 @@ else
 fi
 
 if echo "{{ cookiecutter.enable_heroku_deployment }}" | grep -iq "^n"; then
-    rm -rf uwsgi.ini Procfile runtime.txt bin
+    rm -rf uwsgi.ini Procfile runtime.txt bin/post_compile
```

Comment:
Whats the reason for doing this ?

Reply:
@tucosaurus `bin` directory only had one file `post_compile` which is used in Heroku deployment.
Earlier when heroku is disabled, it removes the whole `bin` directory but now we have moved the db_schema generation file to the same directory. So, instead of deleting whole directory it just deletes the `post_compile` file